sdk/keyvault/keyvault-keys/README.md

```diff
@@ -641,7 +641,7 @@ async function main() {
   let digest = hash.update(signatureValue).digest();
   console.log("digest: ", digest);
 
-  const signResult = await cryptographyClient.signData("RS256", digest);
+  const signResult = await cryptographyClient.sign("RS256", digest);
```

Comment:
Note that this section is indeed the "Sign" section indeed: https://github.com/Azure/azure-sdk-for-js/pull/11601/files#diff-1c2d798c10ee336ec71c67fc29c2222cR617-R619

Reply:
Also note that these methods accept the same parameters. Here's a test: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/test/public/crypto.spec.ts#L110